__tests__/app.test.js

```diff
@@ -228,4 +228,47 @@ describe('app', () => {
         });
     });
   });
+
+  describe('POST /api/articles/:article_id/comments', () => {
+    it('status: 201, responds with posted comment', () => {
+      const newComment = {
+        username: 'rogersop',
+        body: 'An SQL query walks into a bar, walks up to two tables and asks "Can I join you?"',
+      };
+      return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({ body: { newComment } }) => {
```

Comment:
Would prefer for the new comment to come back on a key of `comment` - with responses we like the keys to be as simple and consistent as possible